rtb/analisi_dei_requisiti_ver0.4.0.typ

```diff
@@ -0,0 +1,732 @@
+    - Titolo del grafico.
+    - Etichetta dell'asse delle ascisse.
+    - Etichetta dell'asse delle ordinate.
+    - Etichette dei valori sugli assi.
+    - Dati del grafico.
+  // TODO: AGGIORNATO IN AUTOMATICO ALL'ARRIVO DI NUOVI DATI?
+- *Inclusioni*:
+  + Visualizzazione titolo del grafico (#link(<uc37>)[UC37]).
+  + Visualizzazione etichetta asse delle ascisse e relativi valori (#link(<uc38>)[UC38]).
+  + Visualizzazione etichetta asse delle ordinate e relativi valori (#link(<uc39>)[UC39]).
+  + Visualizzazione dati sul grafico (#link(<uc40>)[UC40]).
+
+=== UC37 - Visualizzazione titolo del grafico <uc37>
+- *Attore principale*: Amministratore.
+- *Precondizioni*: l'amministratore è autenticato al sistema ed ha accesso alla sezione dedicata ai grafici.
+- *Postcondizioni*: l'amministratore visualizza il titolo del relativo grafico.
+- *Trigger*: l'amministratore desidera prendere visione del titolo del grafico.
+- *Scenario principale*:
+  + L'amministratore accede alla sezione apposita dei grafici.
+  + Viene esposto il titolo del relativo grafico.
+
+=== UC38 - Visualizzazione etichetta asse delle ascisse e relativi valori <uc38>
+- *Attore principale*: Amministratore.
+- *Precondizioni*: l'amministratore è autenticato al sistema ed ha accesso alla sezione dedicata ai grafici.
+- *Postcondizioni*: l'amministratore visualizza le etichette dell'asse delle ascisse e dei valori dell'asse del relativo grafico.
+- *Trigger*: l'amministratore desidera prendere visione delle etichette dell'asse delle ascisse e dei rispettivi valori.
+- *Scenario principale*:
+  + L'amministratore accede alla sezione apposita dei grafici.
+  + Viene esposta l'etichetta dell'asse delle ascisse del relativo grafico.
+  + Vengono esposte le etichette dei valori segnati sull'asse delle ascisse del relativo grafico.
+
+=== UC39 - Visualizzazione etichetta asse delle ordinate e relativi valori <uc39>
+- *Attore principale*: Amministratore.
+- *Precondizioni*: l'amministratore è autenticato al sistema ed ha accesso alla sezione dedicata ai grafici.
+- *Postcondizioni*: l'amministratore visualizza le etichette dell'asse delle ordiante e dei valori dell'asse del relativo grafico.
+- *Trigger*: l'amministratore desidera prendere visione delle etichette dell'asse delle ordinate e dei rispettivi valori.
+- *Scenario principale*:
+  + L'amministratore accede alla sezione apposita dei grafici.
+  + Viene esposta l'etichetta dell'asse delle ordinate del relativo grafico.
+  + Vengono esposte le etichette dei valori segnati sull'asse delle ordinate del relativo grafico.
+
+=== UC40 - Visualizzazione dati sul grafico <uc40>
+- *Attore principale*: Amministratore.
+- *Precondizioni*: l'amministratore è autenticato al sistema ed ha accesso alla sezione dedicata ai grafici.
+- *Postcondizioni*: l'amministratore visualizza i dati sul relativo grafico. Viene mostrato il valore esatto di un dato immediatamente vicino al punto nel grafico.
```

Comment:
Non ho capito: il valore sta nel grafico. Te vuoi visualizzare in maniera precisa x e y? Perché se così, buona fortuna a mettere i valori esatti nel grafico se te ne butto su in db una discreta quantità